### ISSUE TITLE
Change swarm name to P2PCore

### ISSUE THREAD

**Issue by flavioribeiro** [streamroot/clappr-p2phls-plugin#121]

**Comment by shankardevy** [comment]
@flavioribeiro do you mean 'Swarm' class name to be changed to P2PCore?

**Comment by flavioribeiro** [comment]
Yes @shankardevy, I'm not sure yet about this issue but I think that swarm is carring a lot of logics. I have plans to break it in other modules so I think that the first step is change its name to P2PCore. What do you think?

**Comment by shankardevy** [comment]
@flavioribeiro can you close this issue when you get time? I'm seeing this issue always on my issue queue because this is open and I'm mentioned.
